app/src/main/java/org/covidwatch/android/ui/menu/MenuFragment.kt

```diff
@@ -44,4 +55,12 @@ class MenuFragment : Fragment(R.layout.fragment_menu) {
         val browserIntent = Intent(Intent.ACTION_VIEW, Uri.parse(url))
         startActivity(browserIntent)
     }
-}
\ No newline at end of file
+
+    private fun makeTestExposureNotification() {
+        //This was necessary because otherwise a parameter to the constructor was required
+        val exposureInformationRepository: ExposureInformationRepository by inject()
```

Comment:
Ideally, we need to have a `MenuViewModel` and use `ExposureInformationRepository` inside of it. Look at the other Fragments and ViewMode in the app to see how they interact.

Reply:
Is this really necessary?

Reply:
In a good application with a good architecture yes,
But this class might disappear in the future so let's leave for now.

Reply:
I forgot to mention that we need another modification here.
Leaving this code without `MenuViewModel` is okay but still should move
`val exposureInformationRepository: ExposureInformationRepository by inject()` 
to the property of `MenuFragment`. 
So it will be declared in the beginning of the class.